srcs/incs/FDsets.hpp

```diff
@@ -6,20 +6,24 @@
 class FDsets {
 	public:
 		FDsets();
-		void		removeFromReadSet(int fd);
-		void		removeFromWriteSet(int fd);
 		void		addToWriteSet(int fd);
 		void		addToReadSet(int fd);
+		void		removeFd(int fd);
+		void		removeFromWriteSet(int fd);
+		int			getMaxSocket() const;
 		fd_set		*getReadSet();
 		fd_set		*getWriteSet();
 		bool		isReadSet(int fd) const;
 		bool		isWriteSet(int fd) const;
 
 	private:
+		void		removeFromReadSet_(int fd);
+
 		fd_set		read_set_;
 		fd_set		tmp_read_set_;
 		fd_set		write_set_;
 		fd_set		tmp_write_set_;
+		int			max_fd_;
```

Comment:
I like that you add the `max_fd_` here. I wanted to comment about this on the previous PR.